### PR TITLE
Doc/gen edits wed am

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: pyaedt/modules/SolveSetup.py [hunk 1]
```diff
@@ -235,12 +235,14 @@ def enable_expression_cache(self, expressions, report_type="Fields", intrinsics=
...
             is supplied, a corresponding list of intrinsic functions must be supplied.
-        isconvergence, bool, str, or list, optional
+        isconvergence, bool or str or list, optional
             Whether the expression is in the convergence criteria. The default is ``True``.
             If a list of expressions is supplied, a corresponding list of Boolean values must be
             supplied.
         isrelativeconvergence : bool, optional
             The default is ``True``.
+        conv_criteria :
+            The defaut is ``1``.
```
Comment (PipKat) [inline pyaedt/modules/SolveSetup.py:245]:
```suggestion
            The default is ``1``.
```

File: doc/source/API/Post.rst [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
-This module provides all functionalities for creating and editing
-plots in the 3D tools and includes the following classes:
+This section lists modules for creating and editing
+plots in the 3D tools:
 
 .. note::
-   The ``AdvancedPostProcessing`` module requires Python 3 and
-   requires ``numpy``, ``matplotlib``, and ``pyvista``.
+   The `AdvancedPostProcessing` module requires Python 3 and
+   requires NumPy, Matplotlib, and PyVista.
```
Comment (akaszynski) [inline doc/source/API/Post.rst:8]:
```suggestion
   requires `NumPy <https://numpy.org/doc/stable/>`_, `Matplotlib <https://matplotlib.org/>`_, and `PyVista <https://docs.pyvista.org/>`_.
```

File: pyaedt/modules/AdvancedPostProcessing.py [hunk 3]
```diff
@@ -85,6 +85,11 @@ def nb_display(self, show_axis=True, show_grid=True, show_ruler=True):
         show_ruler : bool, optional
             Whether to show the ruler. The default is ``True``.
        
+       Returns
+       -------
+       type
+           Jupyter Notebook display.
```
Comment (akaszynski) [inline pyaedt/modules/AdvancedPostProcessing.py:91]:
```suggestion
       :class:`IPython.core.display.Image`
           Jupyter notebook image.
```

File: pyaedt/modules/AdvancedPostProcessing.py [hunk 4]
```diff
@@ -941,7 +946,7 @@ def animate_fields_from_aedtplt(self, plotname, plot_folder=None, meshplot=False
         """Generate a field plot to an image file (JPG or PNG) using PyVista.
         
         .. note::
-           The PyVista method rebuilds the mesh and overlap fields on the mesh.
+           The PyVista module rebuilds the mesh and the overlap fields on the mesh.
```
Comment (akaszynski) [inline pyaedt/modules/AdvancedPostProcessing.py:949]: Good catch with method vs. module!

File: pyaedt/modules/PostProcessor.py [hunk 5]
```diff
@@ -943,11 +946,11 @@ def export_field_file(self, quantity_name, solution=None, variation_dict=None, f
         phase : str, optional
             Field phase. The default is ``None``.
         sample_points_file : str, optional
-            Name of a file with sample points. The default is ``None``.
+            Name of the file with sample points. The default is ``None``.
         sample_points_lists : list, optional
-            List of sample  points. The default is ``None``.
+            List of the sample  points. The default is ``None``.
```
Comment (akaszynski) [inline pyaedt/modules/PostProcessor.py:951]:
```suggestion
            List of the sample points. The default is ``None``.
```

File: pyaedt/modules/SetupTemplates.py [hunk 6]
```diff
@@ -48,7 +48,7 @@
 
 HFSSTransient = [("Frequency", "5GHz"), ("MaxDeltaS", 0.02), ("MaximumPasses", 20), ("UseImplicitSolver", True),
                  ("IsEnabled", True), ("MeshLink", meshlink), ("BasisOrder", -1), ("Transient", transienthfss)]
-"""HFSS transient analysis setup properties and default values."""
+"""HFSS transient setup properties and default values."""
```
Comment (akaszynski) [inline pyaedt/modules/SetupTemplates.py:51]:
```suggestion
"""HFSS transient analysis setup properties and default values."""
```
I think we should keep analysis.

File: pyaedt/modules/SolveSetup.py [hunk 7]
```diff
@@ -22,13 +22,13 @@ class Setup(object):
...
     parent: str
-        AEDT module for analysis setup.
+        Inherited parent object.
     solutiontype: 
         Type of the setup.
     setupname: str, optional
         Name of the setup. The default is ``"MySetupAuto"``.
     isnewsetup: bool, optional
-        Whether to create the new setup from a template. The default is ``True.``
+        Whether to create the setup from a template. The default is ``True.``
```
Comment (akaszynski) [inline pyaedt/modules/SolveSetup.py:31]:
```suggestion
        Whether to create the setup from a template. The default is ``True``.
```

File: pyaedt/modules/SolveSetup.py [hunk 8]
```diff
@@ -562,7 +564,7 @@ def _setup(self, soltype, arg, newsetup=True):
                 self.omodule.EditAMIAnalysis(self.name,arg)
 
             else:
-                print("Not Implemented Yet")
+                print("Not implemented yet.")
```
Comment (akaszynski) [inline pyaedt/modules/SolveSetup.py:567]: @maxcapodi78, this and the other `print("Not Implemented Yet")` above this should be:
```suggestion
                raise NotImplementedError("Solution type '{}' is not implemented yet".format(soltype))
```
Do you see any issue with this?

**Review by akaszynski — APPROVED** [review]
Approved with minor suggestions.  Please wait until we hear back from Massimo.

File: pyaedt/modules/SetupTemplates.py (hunk 6 above)
Comment (PipKat) [reply pyaedt/modules/SetupTemplates.py:51]: None of the others have "analysis" in them, which is why I removed it (and will leave it out) here.

File: pyaedt/modules/PostProcessor.py [hunk 9]
```diff
@@ -1245,12 +1248,13 @@ def export_model_picture(self, dir=None, name=None, picturename=None, show_axis=
         """Export a snapshot of the model to a JPG file. 
         
         .. note::
-           The function works only in graphical mode.
+           This function works only when AEDT is running in the graphical mode.
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:1251]:
```suggestion
           This method works only when AEDT is running in the graphical mode.
```

**Pull request merged**